### PR TITLE
Saleor 1740 tests for draft orders

### PR DESCRIPTION
**Pull request by karola312**
I want to merge this change because I created tests for draft orders

**PR intended to be tested with API branch:** 

### Pull Request Checklist

1. [ ] All visible strings are translated with proper context.
1. [ ] All data-formatting is locale-aware (dates, numbers, and so on).
1. [ ] Translated strings are extracted.
1. [ ] Number of API calls is optimized.
1. [ ] The changes are tested.
1. [ ] Data-test are added for new elements.
1. [ ] Type definitions are up to date.
1. [ ] Changes are mentioned in the changelog.
1. [ ] The changes are tested in different browsers (Chrome, Firefox, Safari).
1. [ ] The changes are tested in light and dark mode.

### Test environment config

API_URI=https://master.staging.saleor.cloud/graphql/

### CODE REVIEW

**Comment by patrys** [comment]
Task linked: [SALEOR-1740 Tests for Orders -> Drafts](https://app.clickup.com/t/2549495/SALEOR-1740)

File: cypress/utils/channelsUtils.js [hunk 1]
```diff
@@ -1,52 +1,50 @@
 import Channels from "../apiRequests/Channels";
 
-class ChannelsUtils {
-  channels = new Channels();
-  createdChannel;
+const channels = new Channels();
```
Comment (krzysztofwolski) [inline cypress/utils/channelsUtils.js:3]: as discussed in previous PRs - avoid module level variables.

**Review by krzysztofwolski — CHANGES_REQUESTED** [review]

File: cypress/integration/orders/draftOrders.js [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+    getDefaultChannel()
+      .then(channel => {
+        defaultChannel = channel;
+      })
+      .then(() => {
+        cy.fixture("addresses");
+      })
+      .then(addresses => {
+        createCustomer(
+          `${randomName}@ex.pl`,
```
[the remark this answers is not in the document]
Comment (karola312) [reply cypress/integration/orders/draftOrders.js:35]: Ok, I'll change it

File: cypress/utils/channelsUtils.js (hunk 1 above)
Comment (karola312) [reply cypress/utils/channelsUtils.js:3]: Yes, I will fix it in resolving conflicts from previous PR

**Review by krzysztofwolski — APPROVED** [review]

**Pull request merged**